pages/examples/intermediate/wallet-messaging.mdx

```diff
@@ -120,7 +120,7 @@ This section presents the entire script in one block, allowing users to easily c
 ### Script Execution
 
     - Open terminal and navigate to the folder where `wallet_messaging.py` is stored.
-    - Run `pip3 install uagents` on your terminal.
+    - Run `pip3 install uagents fetchai-babble` on your terminal.
```

Comment:
This should really be done by installing using the uagents wallet extra (https://github.com/fetchai/uAgents/blob/main/python/pyproject.toml#L43):
```suggestion
    - Run `pip3 install uagents[wallet]` on your terminal.
```